eddy/plugins/ontology-importer/ontology_importer.py

```diff
@@ -161,7 +161,7 @@ def __init__(self, spec, session):
         super().__init__(spec, session)
         self.afwset = set()
         self.vm = None
-        self.space = 150
+        self.space = 250
```

Comment:
Questo mi sa mi era sfuggito quando ho fattorizzato min/max come costanti, in tal caso usiamo la costante altrimenti con tutti queste ripetizioni tra un pò iniziamo a fare confusione.